ultimatepython/data_structures/linkedlist.py

```diff
@@ -0,0 +1,117 @@
+class Node:
+    def __init__(self, dataval):
+        self.dataval = dataval
+        self.nextval = None
+        self.preval = None
```

Comment:
Any variable or attribute that represents more than one English word should be separated as follows:

- `data_val`
- `next_val`
- `prev_val`